api/uaa/refresh_token.go

```diff
@@ -36,6 +51,7 @@ func (client *Client) RefreshAccessToken(refreshToken string) (RefreshToken, err
 			"Content-Type": {"application/x-www-form-urlencoded"},
 		},
 		Body: body,
+		AddClientAuthorization: true,
 	})
```

Comment:
@aeijdenberg according to golang comment here these are not meant to be url encoded https://github.com/golang/go/blob/master/src/net/http/client.go#L335-L343